gf-guide/requirements.md

```diff
@@ -151,4 +164,15 @@ Find all the glyphsets definition and filter lists in the [Glyphsets repository]
 
 ## Stylistic Sets
 
-The Google Fonts API currently does not support the inclusion of OpenType *Stylistic Set* features. Fonts that absolutely need them need to be published as a separate family with the stylistic feature added to the family name.
+The Google Fonts API does not support the inclusion of OpenType Stylistic Set features. Fonts that absolutely need them, need to be published as a separate family with the stylistic feature added to the family name. 
+
+Currently (by June 2022) the API engineer team is working on the support for them. If you would like to publish the font in any case, it is required to assign a name to each stylistic set. If you are using Glyphs font editor, please check the instructions on how to add the names under the [Stylistic Sets Tutorial](https://glyphsapp.com/learn/stylistic-sets)
```

Comment:
@RosaWagner Would you agree with this

Reply:
I don't think we can safely promise they are currently working on it implying the support might come soon unfortunately. I would remove that part.

- about the name: the stylistic sets are available in the downloadable version, but it is subsetted from the version served by the API. For a user experience matter it is required to name the stylistic sets (etc.) 
- about the subfamilies we should specify: the variant glyphs from the stylistic sets should substitute the base glyph from the original family in the new family.

Reply:
> we can safely promise they are currently working on it implying the support might come soon unfortunately.

Ok. It was my understanding from requirements to TSI fonts and the seek to support "expressive"

> - about the subfamilies we should specify: the variant glyphs from the stylistic sets should substitute the base glyph from the original family in the new family.

This is not always completely true or the case. But, are you referring to the necessity of having the right OT fea for them?

Reply:
I am talking about families like Cormorant or Ysabeau; since Small Caps and SS are not supported, and if they are absolue necessity, then they need to be onboarded as separated families.

Reply:
And the separated families need to append the feature name in the family name, and substitute/replace the base glyph by the variant glyph. (ref to this issue for the delay on OT feature support https://github.com/google/fonts/issues/4522)

Reply:
We are talking about different things. Small caps (`smcp` and `c2sc` feas) are not taken or referred to as [Stylistic Sets](https://fonts.google.com/knowledge/glossary/stylistic_sets) (`salt`).
These are the ones I understand are going to be covered and the ones that actually need to have a name to be identified on the menu accessing the features.

Reply:
I know they are not the same, I am talking about 2 different things from the 2 different paragraphs:

1. About onboarding sub-families because opentype features such as SS and SC are not supported: we need to give instructions for font naming and glyphset of the said sub-families. For example, https://fonts.google.com/?query=Cormorant these all come from the same source file. SC is appended to the family name if the subfamily is small caps. If the subfamily feature a stylistic variant instead of original base glyphs, then the SS feature name is appended to the family name. The original text only speaks of SS, but it also concerns the SC sub-families.

2. About the stylistic set in particular; detail added is that they are still part of the family you can download, so for user experience in desktop app, they need a name.

-> For the first paragraph,  It is about the name of the separated font onboarded to give access to the feature (as base glyph).

-> renaming the title to "Opentype Features" instead of "Stylistic Sets" could therefore avoid confusion. And then we could precise exactly which features are subsetted (which I don't know actually).

Reply:
Sorry my first comment was a bit rushed and not clear

Reply:
Yes, as the main title currently is Stylistic Sets, I had only that in mind for both paragraphs the original first and the second just added. 
- The first one to explain situations like Delis which was published with the actual SS as separated sibling families https://fonts.google.com/?query=Delius
- The second paragraph after the nameID 256 issue + Chaeck we reviewed early on the meeting

So yes, we would need to change the title to include both SC and SS under it.
Although I think it should be an entirely new section, but probably in the future.

Reply:
+ confirming the current status of expected support for SS, to not commit to anything that will not going to happen

Reply:
We'll ask Tobias

Reply:
Update in [c5ce6f5](https://github.com/googlefonts/googlefonts.github.io/pull/34/commits/c5ce6f5c31cd3bfc2723e7f2b6ace0931ffd6ece)
I'll merge this PR here to be able to test all that has been changed. Please feel free to make a comment or an edit if you find the need.